packages/markdowndb/README.md

```diff
@@ -1,3 +1,109 @@
-# markdowndb
+# MarkdownDB [![](https://badgen.net/npm/v/@flowershow/markdowndb)](https://www.npmjs.com/package/@flowershow/markdowndb)
 
-Parse markdown files and store them in an SQL database.
+MarkdownDB is a ... that parses your markdown files and saves them to your local SQLite database, so that you can easily import them into your application.
```

Comment:
Suggest something like:

MarkdownDB is a javascript library for working with a "MarkdownDB" i.e. a simple way to treat a directory of markdown files as a database.

Note: this is obviously moving the work into defining a MarkdownDB. Having tried to explain the idea to a few people over the last week or so i think this is the right way to go i.e. define a MarkdownDB and then this *library* is simply a node/js library for working with a markdowndb (it's also the reference API for now etc).

So what's a MarkdownDB? See WIP at https://github.com/flowershow/flowershow/discussions/477